tnu.go

```diff
@@ -35,6 +35,7 @@ func NewTalosUpdater(ctx context.Context, imageTag string, powercycle bool) (*Ta
 	return &TalosUpdater{
 		imageTag:   imageTag,
 		powercycle: powercycle,
+		staged:     staged,
```

Comment:
Need to add `staged` to `NewTalosUpdater`

Reply:
Done